sessions/functionals.qmd

```diff
@@ -235,23 +237,23 @@ usethis::use_package("purrr")
 ```
 
 Since `{purrr}` is part of the `{tidyverse}`, we don't need to load it
-with `library()`. The next step for using the `map()` functional is to
+with `library()`. Before we'll use the `map()` functional, we need to
 get a vector or list of all the dataset files available to us. We will
 return to using the `{fs}` package, which has a function called
-`dir_ls()` that finds files of a certain pattern. In our case, the
-pattern is `user_info.csv`. So, let's add `library(fs)` to the `setup`
-code chunk. Then, go to the bottom of the `doc/learning.qmd` document,
-create a new header called `## Using map`, and create a code chunk below
-that with {{< var keybind.chunk >}}
+`dir_ls()` that finds files of a certain pattern. So, let's add
+`library(fs)` to the `setup` code chunk. Then, go to the bottom of the
+`doc/learning.qmd` document, create a new header called `## Using map`,
+and create a code chunk below that with {{< var keybind.chunk >}}
 
 The `dir_ls()` function takes the path that we want to search
 (`data-raw/mmash/`), uses the argument `regexp` (short for [regular
 expression](https://r4ds.had.co.nz/strings.html#matching-patterns-with-regular-expressions)
 or also `regex`) to find the pattern, and `recurse` to look in all
-subfolders. We'll cover regular expressions more in the next session.
+subfolders. We'll cover regular expressions more in the next session. In
+our case, the pattern is `user_info.csv`, so the code should look like
+this:
 
-```{r list-user-info-files}
-#| filename: "doc/learning.qmd"
+``` {.r filename="doc/learning.qmd"}
```

Comment:
When the `r` in the code chunk has a `.` before it, it tells Quarto to not run it. We actually want to run this code when building the website. So I am reverting this change.

Reply:
Ah, makes sense!